### DISCUSSION TITLE
ABT-6171YY camera with FH8626V100 SoC

### DISCUSSION THREAD

**Discussion by skymansh** [roleoroleo/yi-hack-MStar#523] [Q&A] [answered]
Hello everyone.
I have a ABT-6171YY 1080p DOME PTZ camera with FH8626V100 SoC, sn: 20220524220621, firmware 5.0.00.00_202203291024.
It works with Yi IoT app and storage videos on microSD card or Cloud.
Can I update with yi-hack-v5 firmware ?
I have try to do a backup with this procedure but it does not work.
https://github.com/roleoroleo/yi-hack-Allwinner/wiki/Dump-your-backup-firmware-(SD-card)
If is not working, can you guide me to a good firmware?
Thank you.

**Comment by roleoroleo** [comment, marked as answer]
Sorry but this platform is not supported.

**Comment by geckoret** [comment]
Try changing the name of config.sh to write_hw_gpio.sh, it worked for me with an Antela camera, same hardware.

**Reply by Itstommy10** [reply]
Hi, I have the same camera as you. Did you manage to install a version of Yi hack? If so, which one?
